upload-to-pypi.sh

```diff
@@ -0,0 +1,11 @@
+
+# make sure to increase the "post" number: e.g. from 0.0.1-1 to 0.0.1-2
+
+rm -rf ./dist
+hatchling build
+twine upload -p $PYPI_KEY dist/*
+
+#ppackage/protobuf/bin/protoc -I ./samples --plugin=protoc-gen-grpc=ppackage/grpc/grpc_python_plugin --grpc_out=./output/python/bookclub --python_out=./output/python/bookclub bookclub/book.proto bookclub/member.proto bookclub/bookclub.proto
+
+pip install protoc-polyglot --upgrade
+pip install protoc-polyglot --upgrade
```

Comment:
2 installations?
```suggestion
```

Reply:
yep, jedna nestaci, neptej se me proc. Chova se to tak, ze prvni neudela nic, ale zjisti, ze je na remote nova verze, druha ji pak nainstaluje